lib/commands/new.js

```diff
@@ -104,6 +106,19 @@ module.exports = async (cwd, dir, ctx) => {
     await Helpers.exec('git add package.json README.md', { cwd: dir });
 };
 
+// TODO Give credit
+// bic'd from here: https://github.com/domenic/sorted-object/blob/master/lib/sorted-object.js
+// slightly tweaked and stylized per hapi linting, changed to accept a custom sorting function
```

Comment:
The code you adapted is WTFPL, so I think this is sufficient.  What do you think?
```suggestion
// Bic'd and adapted from domenic/sorted-object (WTFPL)
```

Reply:
🍺 👍 mm!